contents/handbook/people/feedback.md

```diff
@@ -26,34 +26,19 @@ With us growing the team, we will be splitting the session into smaller groups i
 - That being said, avoid piling on and repeating feedback others have given unless you have a different perspective or can add more context. It is ok to say "+1 to what X said about Y" and move on. Do not spend 2min repeating the same point that has already been made by someone else. 
 - Everyone is responsible for noting down and actioning their own feedback (ie. the people team won't do this for you). 
 
-## Performance reviews
+## Career chats
 
-In addition to informal day-to-day feedback and the full team 360 degree feedback session, it is important that we enable team members to take a step back every so often and look at their performance and aspirations in a wider context. This helps us to support a team member's growth and ensure it is aligned with PostHog's needs. 
+It's difficult for managers to take a step back in weekly 1:1s and talk about long term career goals. The manager is responsible for making sure a calendar invite exists that repeats every six months as a reminder to do this.
 
-This process is intended to be self-serve. The People team will ensure the process is kicked off and recorded properly, but it is the individual team member's responsibility to run the process. If you need support, ask your manager, Coua or Charles for help. 
+The process is as follows:
```

Comment:
The process can be simpler as I recently created [a template](https://github.com/PostHog/meta/blob/main/.github/1-1-TEMPLATES/quarterly-1-1-career-checkin.md) for exactly this! I think it's better as it already includes feedback from managers at PostHog about questions they've found useful. 

So it could literally be:
1. Manager schedule the meeting
2. Get team member* to fill out the template
3. Have the meeting; repeat every ~6 months

I don't think we need the 'gathering feedback from peers' bit as we already have 360 feedback sessions, so this seems repetitive? (I guess people could always optionally ask for feedback if they want more). 

* I prefer 'team member' to 'managee'

Reply:
@charlescook-ph Okay I've removed this section on this page and made some changes to https://posthog.com/handbook/company/management